app/forms/collection_rights_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# This represents the form in the modal on the item page that appears when you
+# click "Set rights" in the sidebar
+class CollectionRightsForm < AccessForm
+  # @param [Cocina::Models::Collection] model
```

Comment:
As above, add YARD to cover the `default_rights` param

Reply:
added